08-kubernetes-sealed-secrets/README.md

```diff
@@ -96,28 +96,30 @@ helm search repo sealed-secrets
 The output looks similar to:
```

Comment:
Before doing any search, make sure to update the local cache for the Helm repositories. When `helm search repo <repo_name>` runs, Helm will query the local cache for metadata. You can update the local cache via:

```shell
helm repo update sealed-secrets
```

On the other hand, if running `helm repo update` without specifying any repository, Helm will update all the cached metadata for repositories.

Reply:
Added a new step with the command that will help the developer to update his helm repo